src/components/aelin/GasSelector.tsx

```diff
@@ -190,7 +190,7 @@ const GasSelector = ({
       nGasPrice = gasPrices[gasSpeed] as number
     } else {
       const gasPrice = gasPrices[gasSpeed] as Eip1559GasPrice
-      nGasPrice = Number(gasPrice.maxFeePerGas.add(gasPrice.maxPriorityFeePerGas))
+      nGasPrice = Number(gasPrice.maxFeePerGas)
```

Comment:
@0xlinus Not sure about this one, but I think that maxPriorityFeePerGas is already included in maxFeePerGas